modules/nf-core/trimgalore/main.nf

```diff
@@ -4,8 +4,8 @@ process TRIMGALORE {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/trim-galore:0.6.10--hdfd78af_0' :
-        'biocontainers/trim-galore:0.6.10--hdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/trim-galore%3A0.6.10--hdfd78af_1' :
```

Comment:
```suggestion
        'https://depot.galaxyproject.org/singularity/trim-galore:0.6.10--hdfd78af_1' :
```
why this change?

Reply:
coming from [here](https://github.com/nf-core/modules/commit/8c5eeedd45e295fc9a4f164631da6a8b37e6b9c6)